### DISCUSSION TITLE
DPMPP 2m sde 1 sample per step rather than 2?

### DISCUSSION THREAD

**Discussion by kalkal11** [Comfy-Org/ComfyUI#1709] [Q&A]
Just digging a bit further into the technically side of things and I've noticed that dpmpp 2m sde appears to be using a single sample per iteration - this is in contrast with how automatic1111 webui seems to operate. 

Perhaps I'm misunderstanding the code and there are in fact 2 samples?

However if not, why is there such a dramatic different for this implementation of the sampler between comfy and auto?

**Comment by kalkal11** [comment]
![image](https://github.com/comfyanonymous/ComfyUI/assets/110500565/507c30c8-034f-4fbc-a9ed-100ce3e65d47)

Just talking to a friend about it, should the 2m variant not consistently be 2 samples per iteration?

https://arxiv.org/pdf/2211.01095.pdf

**Comment by comfyanonymous** [comment]
I'm pretty sure both of us use the exact same code from k diffusion for this sampler.
